server/src/main/java/org/opensearch/index/codec/composite/Composite99DocValuesWriter.java

```diff
@@ -95,23 +112,90 @@
             fieldProducerMap.put(field.name, valuesProducer);
             compositeFieldSet.remove(field.name);
         }
+        segmentFieldSet.remove(field.name);
```

Comment:
`createCompositeIndicesIfPossible` may not be invoked if we don't have any sorted numeric field in the code. Can we handle it in constructor?

Reply:
As discussed, we don't have to call createCompositeIndices if none of the fields are present. 

We probably need to handle in query side if the fields are configured and none of the fields have actual values.

These should be the default values
```
value_count = 0
sum = 0.0
min = null
max = null
avg = null
```